modules/ROOT/pages/afm-monitoring-private-apis.adoc

```diff
@@ -0,0 +1,53 @@
+= Monitoring the Endpoints of Private APIs
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+:imagesdir: ../assets/images
+
+Private APIs are APIs with endpoints that are accessible only within your network. API Functional Monitoring lets you monitor private APIs if you have a subscription to Anypoint Virtual Private Cloud (VPC). With VPC, you can run workers in CloudHub in a virtual, private, and isolated network segment, rather than in a region in which resources are shared.
+
+A _private location_ is a worker that runs in an instance of VPC that is configured to allow workers to access your private network. AFM can create workers within VPC that can run tests against APIs that are accessible only within the network that VPC is configured for. The APIs must be deployed in CloudHub. You can create multiple private locations in a single VPC.
+
+The tests in monitors are run on schedules. For a private location to run according to a schedule, a schedule must be associated with it. A private location can be associated with more than one schedule for more than one monitor.
+
+Before you can run tests from a private location, you must create one, which involves specifying a name and an environment for the worker to run it. The environment must be associated with a VPC and must be the same environment that an API to be tested is deployed in. For example, you might create a private location that you name `myLocation` and set it to run in environment `myEnvironment`. When you choose to run a test in `myLocation`, AFM allocates a worker in `myEnvironment` in Anypoint Virtual Private Cloud.
+
+The following diagram depicts the types of relationship that can exist between monitors, private locations, APIs, and environments. It shows five monitors created in AFM. It also shows a VPC in CloudHub in which there are two associated environments: Environment A and Environment B. In Environment A, there are four private locations and seven APIs being tested. In Environment B, there are one private location and three APIs to be tested. The different line types depict five relationship types.
+
+.The possible relationship types between monitors, schedules, private locations, APIs, and environments
+image::afm-private-locations.png[]
+
+* *First relationship type (represented by solid black arrows):*
++
+Monitor 1 runs on a single schedule. The schedule is associated with Private Location 1. The monitor is written to test one or more endpoints in private API 1 and in private API 2, though it could also test endpoints in more private APIs. The private location and the APIs are all in a single environment.
```

Comment:
GLOBALLY change though > although throughout this list; individual instances not marked.